charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-alpha01/templates/deployment.yaml

```diff
@@ -15,17 +15,12 @@ spec:
       - name: tls
         secret:
           secretName: rancher-webhook-tls
-      {{- if .Values.global.hostNetwork }}
-      hostNetwork: true
-      {{- end }}
```

Comment:
This is weird, unrelated ?

Reply:
It happens because the upstream updated to 0.2.0-alpha02, but my PR still use 0.2.0-alpha01

It is fixed after I rebase the PR.